java/arcs/core/storage/Reference.kt

```diff
@@ -31,13 +34,25 @@ data class Reference(
     val storageKey: StorageKey,
     val version: VersionMap?,
     /** Reference creation time (in milliseconds). */
-    override val creationTimestamp: Long = RawEntity.UNINITIALIZED_TIMESTAMP,
+    private var creationTimestamp_: Long = RawEntity.UNINITIALIZED_TIMESTAMP,
     /** Reference expiration time (in milliseconds). */
-    override val expirationTimestamp: Long = RawEntity.UNINITIALIZED_TIMESTAMP
+    private var expirationTimestamp_: Long = RawEntity.UNINITIALIZED_TIMESTAMP
```

Comment:
These should use underscore as a prefix, not a suffix. See https://kotlinlang.org/docs/reference/coding-conventions.html#property-names "Names for backing properties" section